torch/testing/_internal/common_nn.py

```diff
@@ -4230,6 +4238,11 @@ def padding3d_circular(input, pad):
         target_fn=lambda: torch.empty(15).uniform_().mul(10).floor().long(),
         desc='weights',
     ),
+    dict(
+        module_name='CrossEntropyLossWithSoftLabels',
+        input_size=(15, 10),
+        target_fn=lambda: torch.empty(15, 10).uniform_(),
+    ),
```

Comment:
Does this need a reference_fn?

Reply:
For these, I just copied & tweaked the corresponding CrossEntropy tests.

There's a `reference_fn` defined for the entry with `check_sum_reduction=True`, and that one does strictly more than this entry, so I'll just delete this one.